### ISSUE TITLE
Pictures and Screenshots Fail to Steam

### ISSUE THREAD

**Issue by Ralage86** [Yellow-Dog-Man/Resonite-Issues#2828]
### Describe the bug?

When I press the "Save to Steam" button on my radial while holding a picture, the sound effect plays but the picture does not save to Steam.

### To Reproduce

Not sure if there is a method for reproducing this. I run the Windows build on Linux (Ubuntu 22.04) with Proton 9.

### Expected behavior

Obvious. It should work, and save to my screenshots folder on my computer.

### Resonite Version Number

2024.8.5.1341

### What Platforms does this occur on?

Windows, Linux

### What headset if any do you use?

Desktop

### Log Files

[LINUX-PIG-RIG - 2024.8.5.1341 - 2024-08-28 17_55_20.log](https://github.com/user-attachments/files/16789366/LINUX-PIG-RIG.-.2024.8.5.1341.-.2024-08-28.17_55_20.log)

### Reporters

Ralage86

**Comment by Frooxius** [comment]
I'm confused by this, we don't have any "Save to Steam" button. Are you using a mod?

**Comment by Ralage86** [comment]
No, this is a stock game feature. It told me to remove all mods to run the test. I think I am misquoting the button's name too, it's real text is "Export Screenshot", which for me is supposed to save it to Steam.

**Comment by Frooxius** [comment]
I see. I have a few more questions then:

1) Which sound does actually play? The Steam one? Do you get a notification?
2) In Settings under Integrations, is saving screenshots enabled?

**Comment by Ralage86** [comment]
1. The Steam Screenshot sound effect plays, but there is no balloon/notification.
2. Save Screenshots is enabled.

**Comment by Frooxius** [comment]
If the sound plays, that sounds like it makes it to Steam.

Is the screenshot not actually saved? If you go to the game feed or open the screenshots folder, are they not there?

It is possible to disable the notification in Steam, so they could be actually saving, but you're just not getting the notification pop-up.

**Comment by Ralage86** [comment]
The sound effect plays, but no notification pops up and the screenshot does not get to my computer's Steam screenshot folder for Resonite, I've checked. Notifications for screenshots are turned on, and the normal Steam screenshot key works fine (sound+notification+file).

It used to work. I would grab a screenshot in Resonite, select Export Screenshot, and it would play the sound effect, display the notification, and drop the file onto my computer. I have had this problem for many months now.
I would also like to mention that when I press Shift+P or Ctrl+P to take a screenshot in Resonite, I have exactly the same problem: Steam capture sound plays, but no notification and no file.

**Comment by Frooxius** [comment]
Does it work in any other games?

This sounds like it might be an issue with Steam, rather than Resonite, because we haven't changed anything on our end with the integration. Plus if the sound is playing, that sounds like Resonite triggers the save, at which point it's out of our hands.

**Comment by Ralage86** [comment]
I don't have any other games that make a screenshot within the game that I can press a button to export it to Steam.

I have tried reinstalling the Steam package but to no avail. It turns out that I have to delete the entire .steam folder from my /home folder if I want any useful results, and I have a lot in there that I don't want to delete, including Steam configs. Not sure if this is relevant, but this Steam installation has been through three storage mediums.

**Comment by Frooxius** [comment]
Based on that information I'm not sure if this is really any issue in Resonite itself, or if there's something with your setup.

Given the fact that we haven't changed our integration in a long time and that it still plays the sound, I'm more leaning that it might be something specific to your setup.

Without any more information or being able to replicate this with clean setup, I don't think we can do much at this point.

**Comment by Sharkmare** [comment]
> The sound effect plays, but no notification pops up and the screenshot does not get to my computer's Steam screenshot folder for Resonite, I've checked. Notifications for screenshots are turned on, and the normal Steam screenshot key works fine (sound+notification+file).
> 
> It used to work. I would grab a screenshot in Resonite, select Export Screenshot, and it would play the sound effect, display the notification, and drop the file onto my computer. I have had this problem for many months now. I would also like to mention that when I press Shift+P or Ctrl+P to take a screenshot in Resonite, I have exactly the same problem: Steam capture sound plays, but no notification and no file.

Try enabling the option to display a notification, i think i actually remember vaguely that this was an issue.
![image](https://github.com/user-attachments/assets/0998e5fc-98c0-4afb-af66-34a2ea1939b0)

**Comment by shiftyscales** [comment]
@Sharkmare - I just took a finger photo with the Steam screenshot integration setting enabled and it worked as expected and the photo appeared in my Steam library. Can you double-check that setting / toggle it off and on again?

How are you launching Resonite- are you doing so directly through Steam? In VR? Desktop only?
